ajax/libs/startbootstrap-agency/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "startbootstrap-agency",
+  "filename": "js/agency.min.js",
+  "version": "1.0.4",
+  "homepage": "http://startbootstrap.com/template-overviews/agency/",
+  "authors": "David Miller",
+  "description": "A one page HTML theme for agencies created by Start Bootstrap",
+  "keywords": [
+    "bootstrap",
+    "theme"
+  ],
+  "license": "Apache",
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/IronSummitMedia/startbootstrap-agency"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/IronSummitMedia/startbootstrap-agency.git",
+    "basePath": "",
+    "files": [
+      "js/agency.js",
+      "js/contact_me.js",
```

Comment:
Oh, I find `contact_me.js` is also exist in other theme!
It should be used with `jqBootstrapValidation.js` together.
I will revise the commit!